go/utils/publishrelease/install.sh

```diff
@@ -30,84 +30,88 @@ WORK_DIR=
   fi
 
   # Translate aarch64 to arm64, since that's what GOARCH calls it
-  if [ "$ARCH" == "aarch64" ]; then
-    ARCH="arm64"
+  if [ "$ARCH" == 'aarch64' ]; then
+    ARCH='arm64'
   fi
 
-  if [ "$ARCH-$OS" != "x86_64-Linux" -a "$ARCH-$OS" != "x86_64-Darwin" -a "$ARCH-$OS" != "arm64-Darwin" -a "$ARCH-$OS" != "arm64-Linux" ]; then
-    fail "E_UNSUPPOSED_ARCH" "dolt install.sh only supports installing dolt on x86_64, x86, Linux-aarch64, or Darwin-arm64."
+  if [ "$ARCH-$OS" != 'x86_64-Linux' ] && [ "$ARCH-$OS" != 'x86_64-Darwin' ] && [ "$ARCH-$OS" != 'arm64-Linux' ] && [ "$ARCH-$OS" != 'arm64-Darwin' ]; then
+    fail 'E_UNSUPPOSED_ARCH' 'dolt install.sh only supports installing dolt on Linux-x86_64, Darwin-x86_64, Linux-aarch64, or Darwin-arm64.'
   fi
 
-  if [ "$OS" == Linux ]; then
+  if [ "$OS" == 'Linux' ]; then
     PLATFORM_TUPLE=linux
   else
     PLATFORM_TUPLE=darwin
   fi
-  if [ "$ARCH" == x86_64 ]; then
-    PLATFORM_TUPLE=$PLATFORM_TUPLE-amd64
-  elif [ "$ARCH" == arm64 ]; then
-    PLATFORM_TUPLE=$PLATFORM_TUPLE-arm64
+
+  if [ "$ARCH" == 'x86_64' ]; then
+    PLATFORM_TUPLE="$PLATFORM_TUPLE-amd64"
+  else
+    PLATFORM_TUPLE="$PLATFORM_TUPLE-arm64"
   fi
 }
 
 assert_dependencies() {
-  type -p curl > /dev/null || fail "E_CURL_MISSING" "Please install curl(1)."
-  type -p tar > /dev/null || fail "E_TAR_MISSING" "Please install tar(1)."
-  type -p uname > /dev/null || fail "E_UNAME_MISSING" "Please install uname(1)."
-  type -p install > /dev/null || fail "E_INSTALL_MISSING" "Please install install(1)."
-  type -p mktemp > /dev/null || fail "E_MKTEMP_MISSING" "Please install mktemp(1)."
+  type -p curl > /dev/null || fail 'E_CURL_MISSING' 'Please install curl(1).'
+  type -p tar > /dev/null || fail 'E_TAR_MISSING' 'Please install tar(1).'
+  type -p uname > /dev/null || fail 'E_UNAME_MISSING' 'Please install uname(1).'
+  type -p install > /dev/null || fail 'E_INSTALL_MISSING' 'Please install install(1).'
+  type -p mktemp > /dev/null || fail 'E_MKTEMP_MISSING' 'Please install mktemp(1).'
 }
 
 assert_uid_zero() {
-  uid=`id -u`
+  uid="$(id -u)"
   if [ "$uid" != 0 ]; then
-    fail "E_UID_NONZERO" "dolt install.sh must run as root; please try running with sudo or running\n\`curl $INSTALL_URL | sudo bash\`."
+    fail 'E_UID_NONZERO' "dolt install.sh must run as root; please try running with sudo or running\n\`curl $INSTALL_URL | sudo bash\`."
   fi
 }
 
 create_workdir() {
-  WORK_DIR=`mktemp -d -t dolt-installer.XXXXXX`
+  WORK_DIR="$(mktemp -d -t dolt-installer.XXXXXX)"
   cleanup() {
     rm -rf "$WORK_DIR"
   }
+
   trap cleanup EXIT
   cd "$WORK_DIR"
 }
 
 install_binary_release() {
-  local FILE=dolt-$PLATFORM_TUPLE.tar.gz
-  local URL=$RELEASES_BASE_URL/$FILE
-  echo "Downloading:" $URL
-  curl -A "$CURL_USER_AGENT" -fsL "$URL" > "$FILE"
+  local FILE="dolt-$PLATFORM_TUPLE.tar.gz"
+  local URL="$RELEASES_BASE_URL/$FILE"
+
+  echo "Downloading: $URL"
+  curl -a "$CURL_USER_AGENT" -fsL "$URL" > "$FILE"
```

Comment:
This change from `-A` to `-a` doesn't look right to me.

https://curl.se/docs/manpage.html#-A

Reply:
Sorry, my bad on this one. Let me push a quick fix.